package.json

```diff
@@ -11,8 +11,8 @@
     "bump-version": "turbo run bump-version",
     "dev": "turbo run dev",
     "lint": "turbo run lint",
-    "format": "prettier --write \"**/*.{js,ts,tsx,md}\" --cache",
-    "prepare": "husky install",
```

Comment:
We do need the prepare command even with lefthook, just that in this case it should be `lefthook install`

Reply:
Sorry. Didn't knew. Will add it.

Reply:
Done